### DISCUSSION TITLE
Support for Flow

### DISCUSSION THREAD

**Discussion by lanmonster** [statelyai/xstate#1466] [General]
Can we get Flow types? I am constrained to using Flow, but I still want to use XState and have it be type-checked.

**Comment by davidkpiano** [comment]
I don't think it's worth the additional effort to support Flow, to be honest, as most projects that use XState either use plain JavaScript or TypeScript.

I would gladly accept a PR for this, though, but hopefully this can be automated away instead of manually done; e.g., with something like https://github.com/joarwilk/flowgen.
